server_fn/src/default.rs

```diff
@@ -64,6 +64,8 @@ inventory::collect!(DefaultServerFnTraitObj);
 ///     }
 /// }
 /// ```
+///
+/// [`actix-web`]: <https://docs.rs/actix-web/>
```

Comment:
This doesn't seem to produce a working link to `actix-web`'s docs when I run `cargo doc` locally.

Reply:
Fixed. The `lib.rs` file also has a link to actix-web.